packages/components/README.md

```diff
@@ -46,14 +46,15 @@ Let's look at how to write a complex component using the compose framework. This
 
 Each control dedicates file using the format [Button.types.ts](./Button/src/deprecated/Button.types.ts) to define the component's tokens, props, and slots.
 
-Button is essentially a composition of View which is a layout element, a Text element, and an Icon element. The Button's slots reflect this composition. The root slot is a View which will wrap another View, represented as a stack slot which will contain the image and text content.
+Button is essentially a composition of View which is a layout element, a Text element, an Icon element, and a Shadow element. The Button's slots reflect this composition. The root slot is a View which will wrap another View, represented as a stack slot which will contain the image and text content.
```

Comment:
This is talking about and giving code examples for the old Button so I'm not sure if it makes sense to add details about the Shadow here

Reply:
Oh, this isn't for ButtonV1? We should revert this then. I assume the document for ButtonV1 is SPEC.md?

Reply:
Yeah. This is explaining generally how components are built (it's the README in the components folder), but it's outdated and references the old framework.